includes/general.php

```diff
@@ -1052,7 +1052,14 @@ function pods_shortcode_run( $tags, $content = null ) {
 
 		return $pod->form( $tags['fields'], $tags['label'], $tags['thank_you'] );
 	} elseif ( ! empty( $tags['field'] ) ) {
-		if ( empty( $tags['helper'] ) ) {
+		$template = ! empty( $tags['template'] ) ? $tags['template'] : $content;
+		if ( $template ) {
+			$return  = '';
+			$related = $pod->field( $tags['field'], array( 'output' => 'find' ) );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 29d054e3102146a1da4b5c0fc77b220f</sub>